spec/honeycomb/integrations/rails_spec.rb

```diff
@@ -43,51 +39,127 @@
 
     class TestController < ActionController::Base
       def hello
-        render plain: "Hello World!"
+        render plain: "Hello #{params[:name]}!"
+      end
+    end
+
+    shared_examples_for "the rails integration" do
+      let(:event_data) { libhoney_client.events.map(&:data) }
+
+      it_behaves_like "event data"
```

Comment:
it would be great to include the http fields checks here as well! We would probably need to include some additional headers in a `before` block to satisfy the conditions though
```suggestion
      it_behaves_like "event data", http_fields: true
```